s3transfer/manager.py

```diff
@@ -42,14 +45,31 @@ def __init__(self,
         :param max_queue_size: The maximum amount of requests that
             can be queued at a time. A value of zero means that there
             is no maximum.
+
+        :param max_io_queue_size: The maximum amount of read parts that
+            can be queued to be written to disk per download. A value of zero
+            means that there is no maximum.
+
+        :param num_download_attempts: The number of download attempts that
+            will be tried upon errors with downloading an object in S3.
```

Comment:
Can we update the docs to say more retries can happen based on if they happen while writing the file out to disk vs. in the initial service response.  For each GetObject attempt in s3transfer, botocore could retry things like 5xx responses N times before handing the response over to s3transfer.

Not sure exactly how to word it, but something that lets them know there's more retries happening depending on where errors happen.